Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -15,7 +15,26 @@ public struct Initialize: ParsableCommand {
     name: [.long, .customShort("n")],
     help: "Name used to scope the generated schema type files."
   )
-  var schemaName: String = ""
```

Comment:
This isn't needed - swift argument parser will give a meaningful error to the user when a parameter isn't supplied when it's definition isn't initialized.